### PR TITLE
store: switch to v2/assertions api

### PR DESCRIPTION
**Pull request by stolowski**
Switch to v2/assertions endpoint.
This has small regression potential (not just on our side, but also store - we weren't using this endpoint before).

### CODE REVIEW

File: store/store_asserts.go [hunk 1]
```diff
@@ -45,13 +45,6 @@ func (s *Store) assertionsEndpointURL(p string, query url.Values) *url.URL {
 }
 
 type assertionSvcError struct {
-	// v1 error fields
```
Comment (pedronis) [inline store/store_asserts.go:48]: we can't do this until the store switches to return us v2 endpoints in refresh fetch_assertions, we need to discuss that with Przemysław

Comment (stolowski) [reply store/store_asserts.go:48]: Ah, ok, thanks for pointing this out.

**Review by pedronis — APPROVED** [review]
this looks fine, but we have the issue that we might not be able to land this before we cut 2.49 :/, let's chat with @mvo on Monday about timing

**Comment by pedronis** [comment]
@mvo5 cut 2.49 now, so this is unblocked

**Comment by mvo5** [comment]
Looks fine

**Pull request merged**